eng/tox/tox.ini

```diff
@@ -94,7 +94,7 @@ commands =
 
 [testenv:pylint]
 description=Lints a package with pylint (version {[testenv:pylint]pylint_version})
-pylint_version=3.2.7
+pylint_version=3.3.6
```

Comment:
I actually dont want to update the pylint versions, I want to update the azure-pylint-guidelines-checker version and update it in documentation in the repo